lib/features/rewards/helpers/line_draw.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
```

Comment:
I'm not 100% sure, but probably `package:flutter/widgets.dart` would be enough here